tests/cluster/config.py

```diff
@@ -0,0 +1,19 @@
+DOMAIN = '.tn.ixsystems.net'
+NET_INFO = {
+    'CLUSTER': {'ip': '10.234.64.71', 'fqdn': f'tn-kvm01{DOMAIN}'},
+    'NODE_A': {'ip': '10.234.65.10', 'fqdn': f'tn-kvm01-nodea{DOMAIN}'},
+    'NODE_B': {'ip': '10.234.65.11', 'fddn': f'tn-kvm01-nodeb{DOMAIN}'},
+    'NODE_C': {'ip': '10.234.65.12', 'fqdn': f'tn-kvm01-nodec{DOMAIN}'},
+    'NETMASK': 22,
+    'INTERFACE': 'enp1s0',
+    'DEFGW': '10.234.64.1',
+    'DNS1': '10.231.0.2',
+    'DOMAIN': DOMAIN,
+}
```

Comment:
@yocalebo and @pkgdemon, should this be sourced privately?

Reply:
@ericbsd I see no harm with this being public 🤷